HabboHotel/Navigator/NavigatorManager.cs

```diff
@@ -141,4 +144,16 @@ orderby cat.Value.OrderId
                 userId
             })).ToDictionary(search => search.Id);
     }
+
+    public async Task SaveHomeRoom(Habbo habbo, int roomId)
+    {
+        habbo.HomeRoom = roomId;
+        if (roomId == 0)
+            return;

```

Comment:
I thi k it should still be possible to set it to 0 and save

Reply:
so `if (roomId == 0)` enough?